<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ds_sinhvien</class>
 <widget class="QWidget" name="ds_sinhvien">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Điểm danh sinh viên</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/nguye/OneDrive/Hình ảnh/donga.jpg</normaloff>C:/Users/nguye/OneDrive/Hình ảnh/donga.jpg</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>40</y>
     <width>371</width>
     <height>157</height>
    </rect>
   </property>
   <property name="title">
    <string>Thông tin sinh viên</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>23</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mã sinh viên</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maSinhVien">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>23</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>49</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Họ và tên</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hoVaTen">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>49</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Địa chỉ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="diaChi">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>75</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>101</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SĐT</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="soDienThoai">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>101</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>127</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ảnh sinh viên</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="anhSinhVien">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>127</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>371</width>
     <height>56</height>
    </rect>
   </property>
   <property name="title">
    <string>Thao tác</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="them">
      <property name="text">
       <string>Thêm</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="sua">
      <property name="text">
       <string>Sửa</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="xoa">
      <property name="text">
       <string>Xóa</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="timKiem">
      <property name="text">
       <string>Tìm kiếm</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="lamTrong">
      <property name="text">
       <string>Làm trống</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>601</width>
     <height>301</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Mã sinh viên</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Họ tên</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Địa chỉ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Số điện thoại</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ảnh sinh viên</string>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>221</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Ảnh sinh viên</string>
   </property>
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../BTL Python QLTV/PyQT6_Quan_ly_thu_vien/Pictures/Saved Pictures/loading-load.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="soLuongRow">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
